Panic in TestCancelAndReadd (again)

I have started to see #45375 again:

https://github.com/openshift/origin/pull/16071 -> https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/16071/test_pull_request_origin_unit/1869

The longer wait just pushes the race condition forward.

/assign @gmarek

cc @davidopp @timothysc

```
=== RUN   TestCancelAndReadd
panic: sync: negative WaitGroup counter

goroutine 419 [running]:
sync.(*WaitGroup).Add(0xc4201d2420, 0xffffffffffffffff)
	/usr/lib/golang/src/sync/waitgroup.go:75 +0x255
sync.(*WaitGroup).Done(0xc4201d2420)
	/usr/lib/golang/src/sync/waitgroup.go:100 +0x42
github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/controller/node.TestCancelAndReadd.func1(0xc4201c1380, 0x1, 0x20)
	/go/src/github.com/openshift/origin/_output/local/go/src/github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/controller/node/timed_workers_test.go:118 +0x61
github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/controller/node.(*TimedWorkerQueue).getWrappedWorkerFunc.func1(0xc4201c1380, 0x0, 0x0)
	github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/controller/node/_test/_obj_test/timed_workers.go:81 +0xb6
github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/controller/node.CreateWorker.func1()
	github.com/openshift/origin/vendor/k8s.io/kubernetes/pkg/controller/node/_test/_obj_test/timed_workers.go:44 +0x72
created by time.goFunc
	/usr/lib/golang/src/time/sleep.go:170 +0x52
```

--- a/pkg/controller/node/scheduler/timed_workers_test.go
+++ b/pkg/controller/node/scheduler/timed_workers_test.go
@@ -61,7 +61,7 @@ func TestExecuteDelayed(t *testing.T) {
 		return nil
 	})
 	now := time.Now()
-	then := now.Add(time.Second)
+	then := now.Add(3 * time.Second)
 	queue.AddWork(NewWorkArgs("1", "1"), now, then)
 	queue.AddWork(NewWorkArgs("2", "2"), now, then)
 	queue.AddWork(NewWorkArgs("3", "3"), now, then)
@@ -89,7 +89,7 @@ func TestCancel(t *testing.T) {
 		return nil
 	})
 	now := time.Now()
-	then := now.Add(time.Second)
+	then := now.Add(3 * time.Second)
 	queue.AddWork(NewWorkArgs("1", "1"), now, then)
 	queue.AddWork(NewWorkArgs("2", "2"), now, then)
 	queue.AddWork(NewWorkArgs("3", "3"), now, then)
@@ -119,7 +119,7 @@ func TestCancelAndReadd(t *testing.T) {
 		return nil
 	})
 	now := time.Now()
-	then := now.Add(time.Second)
+	then := now.Add(3 * time.Second)
 	queue.AddWork(NewWorkArgs("1", "1"), now, then)
 	queue.AddWork(NewWorkArgs("2", "2"), now, then)
 	queue.AddWork(NewWorkArgs("3", "3"), now, then)

